core/templates/pages/topic-viewer-page/topic-viewer-page.directive.html

```diff
@@ -1,54 +1,97 @@
-           translate="I18N_TOPIC_VIEWER_TRAIN">
-        </a>
+      <li ng-class="{'topic-viewer-tabs-active': $ctrl.activeTab === 'practice'}">
+        <div class="topic-viewer-tabs-text" ng-click="$ctrl.setActiveTab('practice')">
+          <img class="tab-icon" ng-src="<[$ctrl.getStaticImageUrl('/icons/train_icon_24px.svg')]>">
+          <a class="tab-title"
+             translate="I18N_TOPIC_VIEWER_PRACTICE">
+          </a>
+        </div>
       </li>
       <li ng-class="{'topic-viewer-tabs-active': $ctrl.activeTab === 'subtopics'}">
-        <a class="topic-viewer-tabs-text"
-           ng-click="$ctrl.setActiveTab('subtopics')"
-           translate="I18N_TOPIC_VIEWER_REVIEW">
-        </a>
+        <div class="topic-viewer-tabs-text" ng-click="$ctrl.setActiveTab('subtopics')">
+          <img class="tab-icon" ng-src="<[$ctrl.getStaticImageUrl('/icons/review_icon_24px.svg')]>">
+          <a class="tab-title"
+             translate="I18N_TOPIC_VIEWER_REVISION">
+          </a>
+        </div>
       </li>
     </ul>
   </div>
-  <div ng-if="$ctrl.activeTab === 'story'" class="oppia-story-tab">
-    <stories-list canonical-stories-list="$ctrl.canonicalStoriesList"></stories-list>
-  </div>
-  <div ng-if="$ctrl.activeTab === 'practice'" class="oppia-practice-tab">
-    <practice-tab topic-name="$ctrl.topicName"></practice-tab>
-    <skills-mastery-list degrees-of-mastery="$ctrl.degreesOfMastery" skill-descriptions="$ctrl.skillDescriptions"></skills-mastery-list>
-  </div>
-  <div ng-if="$ctrl.activeTab === 'subtopics'" class="oppia-subtopics-tab">
-    <subtopics-list subtopics-list="$ctrl.subtopics"></subtopics-list>
+  <background-banner></background-banner>
+  <div class="topic-viewer-page-content" ng-if="!$ctrl.topicIsLoading">
+    <div ng-if="$ctrl.activeTab === 'info'">
+      <topic-info-tab topic-name="$ctrl.topicName"
+                      topic-description="$ctrl.topicDescription"
+                      story-count="$ctrl.canonicalStorySummaries.length"
+                      subtopic-count="$ctrl.subtopics.length"
+                      chapter-count="$ctrl.chapterCount">
+      </topic-info-tab>
+    </div>
+    <div ng-if="$ctrl.activeTab === 'story'">
+      <stories-list canonical-story-summaries="$ctrl.canonicalStorySummaries"></stories-list>
+    </div>
+    <div ng-if="$ctrl.activeTab === 'practice'">
+      <practice-tab subtopics-list="$ctrl.subtopics"
+                    topic-name="$ctrl.topicName">
+      </practice-tab>
+    </div>
+    <div ng-if="$ctrl.activeTab === 'subtopics'">
+      <subtopics-list subtopics-list="$ctrl.subtopics"
+                      topic-name="$ctrl.topicName"
+                      topic-id="$ctrl.topicId">
+      </subtopics-list>
+    </div>
   </div>
 </div>
 <style>
+  .navbar-light .navbar-container {
+    background: #00645C;
+  }
+
   .oppia-topic-viewer-tabs-container {
+    background-color: #00645C;
+    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Comment:
Remove the quotes here if it is not required.

Reply:
@aks681